src/Util/MissingTexture.cpp

```diff
@@ -0,0 +1,19 @@
+//
+// Created by 黃漢軒 on 2024/2/26.
+//
```

Comment:
I don't think we have this tradition.

Reply:
It's auto-generated by CLion, I'll remove it.

Reply:
Fix on 5d883b1